kwave/options/simulation_execution_options.py

```diff
@@ -1,81 +1,143 @@
-from dataclasses import dataclass
-from typing import Optional, Union
-
-from kwave import PLATFORM, BINARY_PATH
 from kwave.ksensor import kSensor
 from kwave.utils.checks import is_unix
+from kwave import PLATFORM, BINARY_PATH
+import logging
+from dataclasses import dataclass, field
+from typing import Optional, Union
+from pathlib import Path
+import warnings
+
+warnings.filterwarnings("always")
+warnings.simplefilter("always", DeprecationWarning)
+
+
+logger = logging.getLogger(__name__)
+
+DEPRECATION_VERSION = "0.3.7"
 
 
 @dataclass
 class SimulationExecutionOptions:
```

Comment:
I could add a docstring here to describe the parameters better. This would also improve our documentation.